kiwixbuild/dependencies/lzma.py

```diff
@@ -9,9 +9,9 @@ class lzma(Dependency):
     name = 'lzma'
 
     class Source(ReleaseDownload):
-        archive = Remotefile('xz-5.2.7.tar.gz',
-                             '06327c2ddc81e126a6d9a78b0be5014b976a2c0832f492dcfc4755d7facf6d33',
-                             'https://altushost-swe.dl.sourceforge.net/project/lzmautils/xz-5.2.7.tar.gz'
+        archive = Remotefile('xz-5.2.6.tar.gz',
+                             'a2105abee17bcd2ebd15ced31b4f5eda6e17efd6b10f921a01cda4a44c91b3a0',
+                             'https://altushost-swe.dl.sourceforge.net/project/lzmautils/xz-5.2.6.tar.gz'
```

Comment:
Probably it doesn't make sense to include in the PR the commit that switches lzma to 5.2.7 (and the commit message of this commit must be slightly updated)

Reply:
I wanted to make the fallback to `5.2.6` explicit. Ideally, we would move to `5.2.7` but we don't because of compilation issue. By using two commits, it is explicit (and as we will have to update lzma at a moment, better have too much information)